packages/frontend/src/components/ProjectConnection/index.tsx

```diff
@@ -92,10 +91,12 @@ const ProjectForm: FC<Props> = ({
             <SettingsGridCard>
                 <div>
                     {warehouse && getWarehouseIcon(warehouse)}
-                    <LeftPanelTitle>
-                        <H5>Warehouse connection</H5>
-                        <DocumentationHelpButton href="https://docs.lightdash.com/get-started/setup-lightdash/connect-project#warehouse-connection" />
-                    </LeftPanelTitle>
+                    <Flex align="center">
+                        <Title order={5} style={{ marginRight: '0px' }}>
+                            Warehouse connection
+                        </Title>
+                        <DocumentationHelpButton href="https://docs.lightdash.com/get-started/setup-lightdash/connect-project" />
```

Comment:
the href is different here, pls change to : 
`href="https://docs.lightdash.com/get-started/setup-lightdash/connect-project#warehouse-connection"`